.github/CODEOWNERS

```
@@ -124,6 +124,8 @@
 
 /modules/services/clipmenu.nix                        @DamienCassou
 
+/modules/services/devilspie2.nix                      @dawidsowa
```

please add yourself as codeowner of the tests as well:

```suggestion
/modules/services/devilspie2.nix                      @dawidsowa
/tests/modules/services/devilspie2                    @dawidsowa
```